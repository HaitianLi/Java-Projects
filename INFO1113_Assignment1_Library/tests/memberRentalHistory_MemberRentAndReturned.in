add collection books.csv
add member jack hu
rent 100000 111115
RELINQUISH ALL 100000
member history 100000
exit