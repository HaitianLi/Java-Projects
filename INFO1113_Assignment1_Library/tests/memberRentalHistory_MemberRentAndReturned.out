user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)

user: Ending Library process.
